playwright-mobile.config.ts

```
@@ -0,0 +1,54 @@
+  projects: [
+    {
+      name: 'Mobile Safari',
+      use: { ...devices['iPhone 13 Pro'] },
+    },
+    {
+      name: 'Mobile Chrome',
+      use: { ...devices['Pixel 5'] },
+    },
+    {
+      name: 'Mobile Firefox',
+      use: { 
```

ESLint

[error] Delete `·`

(prettier/prettier)